components/forms/sql/create_shared_schema.sql

```diff
@@ -0,0 +1,101 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+CREATE TABLE IF NOT EXISTS addresses_data (
+    guid          TEXT NOT NULL PRIMARY KEY,
+    given_name          TEXT NOT NULL,
+    additional_name     TEXT NOT NULL,
+    family_name         TEXT NOT NULL,
+    organization        TEXT,                -- Company
+    street_address      TEXT,                -- (Multiline)
+    address_level3      TEXT NOT NULL,       -- Suburb/Sublocality
+    address_level2      TEXT NOT NULL,       -- City/Town
+    address_level1      TEXT NOT NULL,       -- Province (Standardized code if possible)
+    postal_code         TEXT NOT NULL,
+    country             TEXT NOT NULL,       -- ISO 3166
+    tel                 TEXT,                -- Stored in E.164 format
+    email               TEXT,
+
+    -- TODO: removed computed fields, need to figure out
+
+    time_created        INTEGER NOT NULL,
+    time_last_used      INTEGER,
+    time_last_modified  INTEGER NOT NULL,
+    times_used          INTEGER NOT NULL DEFAULT 0,
+
+    sync_change_counter INTEGER NOT NULL DEFAULT 1
+);
+
+CREATE TABLE IF NOT EXISTS addresses_mirror (
+    guid          TEXT NOT NULL PRIMARY KEY,
+    given_name          TEXT NOT NULL,
+    additional_name     TEXT NOT NULL,
+    family_name         TEXT NOT NULL,
+    organization        TEXT,                -- Company
+    street_address      TEXT,                -- (Multiline)
+    address_level3      TEXT NOT NULL,       -- Suburb/Sublocality
+    address_level2      TEXT NOT NULL,       -- City/Town
+    address_level1      TEXT NOT NULL,       -- Province (Standardized code if possible)
+    postal_code         TEXT NOT NULL,
+    country             TEXT NOT NULL,       -- ISO 3166
+    tel                 TEXT,                -- Stored in E.164 format
+    email               TEXT,
+
+    time_created        INTEGER NOT NULL,
+    time_last_used      INTEGER,
+    time_last_modified  INTEGER NOT NULL,
+    times_used          INTEGER NOT NULL DEFAULT 0,
+
+    sync_change_counter INTEGER NOT NULL DEFAULT 1
+);
+
+CREATE TABLE IF NOT EXISTS addresses_tombstones (
+    guid            TEXT PRIMARY KEY,
+    time_deleted    INTEGER NOT NULL
+) WITHOUT ROWID;
+
+CREATE TABLE IF NOT EXISTS credit_cards_data (
```

Comment:
I think we should removing the CC table too if we don't see ourselves populating it in the short term - it might mislead the casual reader. It also allows us to side-step the requirements around client-side encryption of the CC number etc. Leaving it in as a comment saying like "we think the CC table might one day look like..." might be fine too.